app/uk/gov/hmrc/transitmovementseisstub/controllers/MessagesController.scala

```diff
@@ -33,6 +33,6 @@ class MessagesController @Inject() (cc: ControllerComponents)(implicit val mater
   def post(): Action[Source[ByteString, _]] = Action(streamFromMemory) {
     request: Request[Source[ByteString, _]] =>
       request.body.runWith(Sink.ignore)
```

Comment:
Do we not want some type of verification here for expected headers etc to ensure that the router is sending the right things through?

Reply:
I think better to verify all the headers send from router to make sure everything passed from router is received at stub or EIS end.

Reply:
Yeah, I don't mind doing that, this was just the minimum viable change.